### PR TITLE
Replaced HTML unique id's with data attributes

### PR DESCRIPTION
**Pull request by artanmyrtolli**
# Description
Replaced duplicate unique ID's with data attributes
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring

# How Has This Been Tested?
- [ ] open localHost
- [x] dev tools

# Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -56,8 +57,11 @@ function generateNewPalette() {
 }
 
 function locked(event){
-	var index = event.target.id
-	if (!index) return;
+	var index = parseInt(event.target.dataset.index);
+	console.log(index);
```
Comment (michaelputnam67) [inline scripts.js:61]: remove this console log

File: scripts.js [hunk 2]
```diff
@@ -56,8 +57,11 @@ function generateNewPalette() {
...
+	if (isNaN(index)) { 
+		//isNaN is a function that checks if the value is NaN. This is to get rid of the console errors when clicking outside the box
```
Comment (michaelputnam67) [inline scripts.js:63]: So this was used to replace line 60: `if (!index) return:`. I like it.

File: styles.css [hunk 3]
```diff
@@ -1,6 +1,7 @@
 * {
 	font-family: 'Varela Round', sans-serif;
 	background-color: #fff5ee;
+	border: 2px solid red;
```
Comment (michaelputnam67) [inline styles.css:4]: Lets make sure we remove the red border.

**Review by michaelputnam67 — APPROVED** [review]
Awesome job Artan. Way to crack that open. I look forward to you explaining it to me.

**Review by sted1994 — APPROVED** [review]
I think Michael covered everything i had to say.

**Pull request merged**